ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PartyManagement.scala

```diff
@@ -28,7 +30,7 @@ final class PartyManagement(session: LedgerSession) extends LedgerTestSuite(sess
       for {
         ledger <- context.participant()
         party <- ledger.allocateParty(
-          partyHintId = Some("PMAllocateWithHint"),
+          partyHintId = Some(Random.alphanumeric.take(10).mkString),
```

Comment:
Please keep the original name and append the randomized string to it.

Reply:
put this back in and added randomized suffix separated by underscore